src/auth/auth.guard.ts

```diff
@@ -7,7 +7,10 @@ import { User } from "src/user/user.entity";
 @Injectable()
 export class JwtAuthGuard extends AuthGuard("jwt") implements IAuthGuard {
 
-	public handleRequest(err: unknown, user: User): any {
+	public handleRequest(err: unknown, user: User, info: Error): any {
```

Comment:
what is err and info? :))

Reply:
asa era semnatura metodei :))  handleRequest<TUser = any>(err: any, user: any, info: any, context: ExecutionContext, status?: any): TUser;